@kiva/kv-components/vue/KvCountdownTimer.vue

```diff
@@ -26,6 +27,10 @@ export default {
 		const interval = ref(null);
 		const timeLeft = ref(null);
 
+		const remainingHours = computed(() => {
+			return Math.floor(timeLeft.value.asHours());
```

Comment:
How does this differ from `.hours()`?

Reply:
Here the answer from documentation

```
As with the other getters for durations, moment.duration().hours() gets the hours (0 - 23).

moment.duration().asHours() gets the length of the duration in hours.
```

Reply:
Makes sense, thanks! Hopefully we can easily replace this with `date-fns` methods later...